charmcraft/commands/build.py

```diff
@@ -26,6 +26,7 @@
 
 from charmcraft.cmdbase import BaseCommand, CommandError
 from .utils import make_executable
+from .jujuignore import JujuIgnore, default_juju_ignore
```

Comment:
I expect flake8 will fail on these...

Reply:
So it was supposed to be "..jujuignore" since it is in the parent directory, but flake8 doesn't complain at that point.